taskchampion/lib/src/replica.rs

```diff
@@ -565,3 +785,179 @@ pub unsafe extern "C" fn tc_replica_free(rep: *mut TCReplica) {
     }
     drop(replica);
 }
+
+#[ffizz_header::item]
+#[ffizz(order = 903)]
+/// Free a vector of ReplicaOp.  The vector may not be used after this function returns and must not be freed
+/// more than once.
+///
+/// ```c
+/// EXTERN_C void tc_replica_op_list_free(struct TCReplicaOpList *oplist);
+/// ```
+#[no_mangle]
+pub unsafe extern "C" fn tc_replica_op_list_free(oplist: *mut TCReplicaOpList) {
+    // SAFETY:
+    //  - items is not NULL
+    //  - items is properly aligned (promised by caller)
+    let items = unsafe { (*oplist).items };
+    // SAFETY:
+    //  - len is not NULL
+    //  - len is properly aligned (promised by caller)
+    let len = unsafe { (*oplist).len };
+    // SAFETY:
+    //  - capacity is not NULL
+    //  - capacity is properly aligned (promised by caller)
+    let capacity = unsafe { (*oplist).capacity };
+    // SAFETY:
+    //  - items, len, and capacity are the unmodified values returned by vec_into_raw_parts
+    //  (promised by caller)
+    unsafe { Vec::from_raw_parts(items, len, capacity) };
```

Comment:
Again preferring to use the PassByValue functionality:
```suggestion
    debug_assert!(!oplist.is_null());
    // SAFETY:
    // - arg is not NULL (just checked)
    // - `*oplist` is valid (guaranteed by caller not double-freeing this value)
    unsafe {
        TCReplicaOpList::take_val_from_arg(
            oplist,
            // SAFETY:
            //  - value is empty, so the caller need not free it.
            TCReplicaOpList::return_val(Vec::new()),
        )
    };
```